src/stactools/core/projection.py

```diff
@@ -1,10 +1,21 @@
+import math
 from copy import deepcopy
-from typing import Any, Dict, List, Optional, Sequence, Union
+from typing import Any, Dict, List, Optional, Sequence, Tuple, Union
 
 import pyproj
 import rasterio.crs
 import rasterio.transform
 
+# Sinusoidal projection parameters derived from Appendix 2, Section 13.1 in:
+# https://modis-fire.umd.edu/files/MODIS_Burned_Area_Collection51_User_Guide_3.1.0.pdf
+# All parameters specified to 12 significant digits so we can round to 11
+# significant digits after using them in computations. This assures millimeter
+# precision at the projection extremeties.
+SINUSOIDAL_SPHERE_RADIUS = 6371007.18100
+SINUSOIDAL_TILE_METERS = 1111950.51977
+SINUSOIDAL_X_MIN = -20015109.3558
+SINUSOIDAL_Y_MAX = 10007554.6779
```

Comment:
So now that I'm looking at it, I'm a _little_ hesitant to bring this functionality all the way up to **stactools**, as it feels very dataset-specific. I'm now leaning towards keeping this in **stactools-packages-modis**, and maybe **stactools-packages-viirs** depends on **modis** to get these values/functions?

Reply:
Agreed. Removed.